charmcraft/commands/store/__init__.py

```diff
@@ -583,3 +608,54 @@ def _get_lib_info(*, full_name=None, lib_path=None):
     return LibData(
         lib_id=libid, api=libapi, patch=libpatch, content_hash=content_hash, content=content,
         full_name=full_name, path=lib_path, lib_name=lib_name, charm_name=charm_name)
+
+
+class CreateLibCommand(BaseCommand):
+    """Create a charm library."""
+    name = 'create-lib'
+    help_msg = "Create a charm library."
+    overview = textwrap.dedent("""
+        Create a charm library.
+
+        It will request a unique ID from Charmhub and bootstrap a
+        template file in the proper local directory.
+
+        It will automatically take you through the login process if
+        your credentials are missing or too old.
+    """)
+
+    def fill_parser(self, parser):
+        """Add own parameters to the general parser."""
+        parser.add_argument(
+            'lib_name', metavar='lib-name',
+            help="The name of the library file (e.g. 'db').")
+
+    def run(self, parsed_args):
+        """Run the command."""
+        lib_name = parsed_args.lib_name
+        valid_chars = set(string.ascii_lowercase + string.digits + '_')
+        if set(lib_name) - valid_chars or lib_name[0] not in string.ascii_lowercase:
+            raise CommandError(
+                "Invalid library name (can be only lowercase alphanumeric "
+                "characters and underscore, starting with alpha).")
+
+        charm_name = get_name_from_metadata()
+        if charm_name is None:
+            raise CommandError(
+                "Can't access name in 'metadata.yaml' file. The 'create-lib' command needs to "
+                "be executed in a valid project's directory.")
+
+        full_name = 'charms.{}.v0.{}'.format(charm_name, lib_name)
+        lib_data = _get_lib_info(full_name=full_name)
+        lib_path = lib_data.path
+        if lib_path.exists():
+            raise CommandError('The indicated library already exists on {}'.format(lib_path))
+
+        store = Store()
+        lib_id = store.create_library_id(charm_name, lib_name)
+
+        lib_path.parent.mkdir(parents=True, exist_ok=True)
+        lib_path.write_text(LIBRARY_TEMPLATE.format(lib_id=lib_id))
+
+        logger.info("Library %s created with id %s.", full_name, lib_id)
+        logger.info("Make sure to add the library file to your project: %s", lib_path)
```

Comment:
How about
```suggestion
        logger.info("Make sure to add the library file to your project: for example, 'git add %s'", lib_path)
```

Reply:
Is the library already part of the project by virtue of its creation? If so, instead of adding to the project, should that word not be replaced with "version control"?

Reply:
Nice, improved.